nmigen_boards/quickfeather.py

```diff
@@ -0,0 +1,86 @@
+import os
+import sys
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.quicklogic import *
+from nmigen_boards.resources import *
+
+
+__all__ = ["QuickfeatherPlatform"]
+
+
+class QuickfeatherPlatform(QuicklogicPlatform):
+    device      = "ql-eos-s3_wlcsp"
+    part        = "PU64"
```

Comment:
According to the nMigen conventions and [toolchain documentation](https://quicklogic-fpga-tool-docs.readthedocs.io/) this field should be called `package` rather than `part`. (I've already fixed it in the nmigen repo.)

Reply:
fixed